es/es-server/src/main/java/org/elasticsearch/index/engine/Engine.java

```diff
@@ -873,6 +873,13 @@ public final CommitId flush() throws EngineException {
      */
     public abstract void rollTranslogGeneration() throws EngineException;
 
+    /**
+     * Force merges to 1 segment
+     */
+    public void forceMerge(boolean flush) throws IOException {
```

Comment:
as this is only used in tests, I'd prefer to use the calling method directly instead